core/controllers/admin_test.py

```diff
@@ -154,19 +154,22 @@ def test_change_rights(self):
 
         # Log in as a moderator.
         self.login(self.MODERATOR_EMAIL)
+        self.assertEqual(self.testapp.get('/').status_int, 302)
```

Comment:
See above -- this whole test isn't really testing for correct redirection (that's not the point), so you don't need this line. Ditto below.